### PR TITLE
fix(docs): update active state for leftnav menu item

### PR DESCRIPTION
**Pull request by youngheart12**
### What does this implement/fix? Explain your changes.
This commit addresses the issue with the active state not being correctly set for the left menu items in the navigation. By adding window.location.pathname in a useEffect hook, the component can now accurately determine the current path and set the appropriate menu item as active. This change ensures that the UI reflects the correct navigation state based on the URL, improving user experience and navigation consistency.
...

### Does this close any currently open issues?
Yes, this change resolves issue #2367 related to the active menu item display in the sidebar navigation.
...

### Any other comments?
N/A
...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.
There are no breaking changes associated with this commit. All existing functionality remains intact.
...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: docs/cypress.config.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const { defineConfig } = require("cypress");
 
 module.exports = defineConfig({
+  projectId: 'u5oh2w',
```
Comment (anuradha9712) [inline docs/cypress.config.js]: Please confirm why do we need to specify `projectId` here?

Comment (youngheart12) [reply docs/cypress.config.js]: reverted this changes in the latest commit

**Review by anuradha9712 — CHANGES_REQUESTED** [review]
@youngheart12  Could you please squash all your commits into a single commit?

**Review by anuradha9712 — CHANGES_REQUESTED** [review]
@youngheart12 please change the commit message something similar to `"fix(docs): update active state for leftnav menu item"`

**Comment by youngheart12** [comment]
@anuradha9712 Suggested changes done

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**